loggo/loggo.py

```diff
@@ -371,15 +371,11 @@ def _represent_return_value(self, response, truncate=140):
         """
         Make a string representation of whatever a method returns
         """
-        representable = (int, float, str, list, set, dict, type(None), bool, tuple)
-        if isinstance(response, representable):
-            return '({})'.format(self._force_string_and_truncate(response, truncate, use_repr=True))
         # some custom handling for request response objects
-        content = getattr(response, 'content', False)
-        if content:
-            return '({})'.format(self._force_string_and_truncate(content.decode('utf-8'), truncate, use_repr=True))
```

Comment:
The `content.decode('utf-8')` here is the reason for this fix. It throws if `content` cant be utf8 decoded.

Reply:
yep, i like your idea here in general, the tuple of representable types was from an earlier time when there were more unstringable errors. ideally now it can go, it was always pretty crap.